test/e2e/backup-restore/backup_restore_test.go

```diff
@@ -159,7 +160,14 @@ func testBackupRestore(t *testing.T, clusterSpec clusterSpec, clusterSize, numEn
 	testKube.Create(restoreSpec)
 
 	// Ensure the restore pod has joined the cluster
-	testKube.WaitForValidRestorePhase(restoreName, namespace, v2.RestoreSucceeded)
+	err = testKube.WaitForValidRestorePhase(restoreName, namespace, v2.RestoreSucceeded)
+	if err != nil {
+		// Known issue with older operands that won't be fixed, skip the test to reduce noise in the test results
+		if strings.Contains(err.Error(), "ISPN-15173") {
+			tutils.SkipPriorTo(t, "14.0.21", "Known issue: "+err.Error())
```

Comment:
Nitpick. This should be the same as the ISPN-15173 fix version "14.0.18" as upstream we support "14.0.19", "14.0.20" and we may want to execute the test with said versions at some point.